pkg/local_object_storage/engine/evacuate_test.go

```diff
@@ -210,6 +227,20 @@ func TestEvacuateNetwork(t *testing.T) {
 		prm.handler = acceptOneOf(objects, totalCount-1)
 
 		res, err := e.Evacuate(prm)
+		if !errors.Is(err, errReplication) {
+			fmt.Println("!!! Connect and check!!!!!")
```

Comment:
What is this for?

Reply:
Sorry, removed. Where was my mind when I push this?)